src/components/FieldNote/FieldNote.stories.tsx

```diff
@@ -26,16 +26,16 @@ export const Default: StoryObj<Args> = {
 export const WithText: StoryObj<Args> = {
   args: {
     children: (
-      <Text as="div" size="sm">
-        <p>Here is a field note that involves:</p>
-        <ul>
-          <li>Multiple lines</li>
-          <li>Arbitrary HTML text</li>
-          <li>
+      <div className="max-w-xl">
+        <Text className="mb-6">Here is a field note that involves:</Text>
+        <ul className="ml-4 list-disc">
+          <li className="mb-0">Multiple lines</li>
+          <li className="mb-0">Arbitrary HTML text</li>
+          <li className="mb-0">
```

Comment:
I would have thought preflight would make resetting the bottom margin unnecessary? Unless this is doing something else...

Reply:
Yup, these classes are redundant, removed